solutions/observability/apm/attributes.md

```diff
@@ -0,0 +1,82 @@
+---
+mapped_pages:
+  - https://www.elastic.co/guide/en/observability/current/apm-open-telemetry-resource-attributes.html
+  - https://www.elastic.co/guide/en/serverless/current/observability-apm-agents-opentelemetry-resource-attributes.html
+applies_to:
+  stack:
+  serverless:
+---
+
+# Attributes and labels [apm-open-telemetry-resource-attributes]
+
+In OpenTelemetry, an attribute is a key-value pair. Attributes are similar to [labels](/solutions/observability/apm/metadata.md#apm-data-model-labels) in that they add metadata to transactions, spans, and other entities.
+
+Resource attributes are a type of attribute that contains information about the entities that produce telemetry. Resource attributes map to Elastic Common Schema (ECS) fields like `service.*`, `cloud.*`, `process.*`, and so on. These fields describe the service and its environment.
+
+For example, Elastic maps the OpenTelemetry `deployment.environment` field to the ECS `service.environment` field on ingestion.
+
+## Setting resource attributes
+
+You can set resource attributes through environment variables or by editing the configuration of the resource processor of the OpenTelemetry Collector.
+
+### OpenTelemetry agent
+
+Use the `OTEL_RESOURCE_ATTRIBUTES` environment variable to pass resource attributes at process invocation. For example:
+
+```bash
+export OTEL_RESOURCE_ATTRIBUTES=deployment.environment=production
+```
+
+### OpenTelemetry Collector
+
+Use the [resource processor](https://github.com/open-telemetry/opentelemetry-collector-contrib/tree/main/processor/resourceprocessor) to set or apply changes to resource attributes when using the OTel Collector.
+
+```yaml
+...
+processors:
+  resource:
+    attributes:
+    - key: deployment.environment
+      action: insert
+      value: production
+...
+```
+
+## Handling of unmapped attributes
+
+When sending telemetry to APM Server, only a subset of OpenTelemetry attributes are directly mapped to ECS fields. If an attribute doesn't have a predefined ECS mapping, the system stores it under `labels.*`, with dots replaced by underscores.
```

Comment:
This line still lacks the part describing unmapped resource attributes are equiv to global labels in Elastic APM agent but unmapped record-level attributes (log record, span, data point) are normal labels.